### ISSUE TITLE
Armorpaint won't launch, installation went smoothly.

### ISSUE THREAD

**Issue by Masgame** [armory3d/armorpaint#1373]
I installed it through git, followed every step carefully and everything worked fine until I tried to open the executable, and nothing. No error messages, it just doesn't launch. It is running off my GPU in the Nvidia control panel. In VS I changed the build options to release and x64 bit. During the build there was this message that might be of concern:

1>Previous IPDB not found, fall back to full compilation.

It's also not installed on C drive, and I have a mobile 1060 6GB. Windows 10.
Any idea what's going on?

**Comment by Caysu** [comment]
It will work if you press the green run button in visual studio, or this answer assumes that anyway. I found it worked when I pressed that, and didn't in the x64/Release directory when I double clicked the ArmorPaint.exe.

I grabbed the launch args of the process that visual studio's run button created, it passes the directory armorpaint/build/krom as an arg to the ArmorPaint.exe. This means the issue can be solved by creating a `.bat` script in the directory `armorpaint/build/x64/Release/`. Right click->`New->Text Document`. Rename the new text document you created to end in the `.bat` extension. I named mine  `run.bat`.

Inside the `.bat` you created, insert the following, ensuring to replace "PATH_TO_CHROM_FOLDER" with the full path to your `armorpaint/build/krom` directory. On Windows, it's easiest to navigate to `armorpaint/build`, then while holding left shift, right click the `krom` directory. In the context menu, an option will appear titled "Copy as path". This will escape/quote the path correctly so it works in the `.bat` script.

```
"ArmorPaint.exe" PATH_TO_CHROM_FOLDER
```

Here's what my `run.bat` looks like:

```
"ArmorPaint.exe" "A:\armorpaint\build\krom"
```

Double click the `.bat` and ArmorPaint should run as usual.
